src/IncludedCommands/Administrative/slock.lua

```diff
@@ -13,44 +13,92 @@ local Types = require(script.Parent.Parent.Parent:WaitForChild("Types"))
 return {
     Keyword = {"slock", "serverlock"},
     Category = "Administrative",
-    Description = "Locks and unlocks the server for non-admins.",
+    Description = "Locks and unlocks the server for non-admins or a minimum AdminLevel.",
     Arguments = {
         {
             Type = "boolean",
             Name = "Active",
             Description = "Whether the server lock is active or not.",
             Optional = true,
         },
+        {
+            Type = "number",
+            Name = "AdminLevel",
+            Description = "Minimum AdminLevel a player must have to join.",
+            Optional = true,
+        },
     },
     ServerLoad = function(Api: Types.NexusAdminApiServer)
-        Api.CommandData.ServerLocked = false
+        Api.CommandData.ServerLocked = nil
```

Comment:
While it was here before, I'd like the `ServerLocked` variable renamed to something like `MinimumAdminLevel`, `CurrentMinimumAdminLevel`, `RequiredAdminLevel`, or something else that makes it clear it is an admin level instead of a bool.